var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -3,14 +3,219 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import glob
 import os
 
-from llnl.util.filesystem import join_path
+from llnl.util.filesystem import find, join_path, working_dir
 
-from spack import *
+import spack.build_environment
+from spack.build_systems.makefile import MakefilePackage
+from spack.directives import *
```

Comment:
This shouldn't be needed in packages, these are all already included

Reply:
True, same for the filesystem ones.  They came in from being needed in the build_system version, but are now gone.